lib/pbench/server/api/resources/users_api.py

```diff
@@ -296,40 +296,48 @@ class UserAPI(Resource):
     Abstracted pbench API to get user data
     """
 
+    # Helper NamedTuple containing a target user, expected http status to return with message
+    TargetUser = NamedTuple(
+        "TargetUser",
+        [("target_user", User), ("http_status", HTTPStatus), ("http_message", str)],
+    )
+
     def __init__(self, logger, auth):
         self.logger = logger
         self.auth = auth
 
     def get_valid_target_user(
         self, target_username: str, request_type: str
-    ) -> NamedTuple:
+    ) -> "TargetUser":
```

Comment:
That's interesting. Did you have problems without the quotes? I would have thought the `TargetUser` definition would have been fully resolved by this point.

Reply:
Flake8 is complaining that it found an undefined name 'TargetUser' if I dont use the quote

Reply:
Presumably you need to qualify it:  `UserAPI.TargetUser`.

Reply:
weird I get `undefined name 'UserAPI'` now

Reply:
Ah, of course; because we _are_ still defining the class `UserAPI`, so that doesn't resolve yet. Sigh.

So maybe it should be `"UserAPI.TargetUser"??